### ISSUE TITLE
Null Pointer Exception

### ISSUE THREAD

**Issue by rarean** [sridharbandi/Java-a11y#6]
Get NPE when using Java-a11y as a dependency jar file. Null pointer is because it cannot find the HTML_CS file and the templates in "ftl" dir. 
Proposed solution: use `rgetResourceAsStream` and `setClassLoader` respectively to fix NPE

**Comment by sridharbandi** [comment]
With @rarean PR, this issue is fixed
